Casks/c/consul.rb

```diff
@@ -13,7 +13,7 @@
 
   livecheck do
     url "https://releases.hashicorp.com/consul/"
-    regex(/href=.*?consul[._-]v?(\d+(?:\.\d+)+)/i)
+    regex(/href=.*?consul[._-]v?(\d+(?:\.\d+)+)\s*</i)
```

Comment:
```suggestion
    regex(%r{href=.*?/consul/v?(\d+(?:\.\d+)+)/}i)
```

We usually  restrict matching to `href` attributes rather than link text.